include/itemlistformaction.h

```diff
@@ -156,8 +156,6 @@ class ItemListFormAction : public ListFormAction {
 	FilterContainer& filters;
 
 	void handle_op_saveall();
-
-	ListWidget items_list;
```

Comment:
Ditto about `#include "listwidget.h"`.

Reply:
Disregard that too (see https://github.com/newsboat/newsboat/pull/1079#discussion_r452113878).